manifests/m/MarkoBL/Rosi/6.0.0/MarkoBL.Rosi.installer.yaml

```
@@ -0,0 +1,20 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=QUSU.7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: MarkoBL.Rosi
+PackageVersion: 6.0.0
```

4 digit Version Number
```suggestion
PackageVersion: 6.0.0.0
```
Also Duplicate of https://github.com/microsoft/winget-pkgs/pull/41675, but noted that it will be automated once the version is corrected 👍